bazelw

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -euo pipefail
+
+read -r bazel_version<.bazelversion
```

Comment:
this doesn't really help if we still want sha validation, since that would be specific to each version

Reply:
I mean it will still hardcode validation against a specific target, but that's still useful right?

Reply:
yea just too bad that you'd still have to change 2 files per update. I think if no one else is really attached to this we should delete it and encourage everyone to use bazelisk instead

Reply:
It's too bad, but I mean, if we do sha validation with a file-based version it's also kind of a given.

Again, I think I do like having the bash script here, that supports the same configuration as bazelisk, giving the option to use either. Would you prefer to drop the sha validation? (I don't view this as a necessity; I'm fine with keeping it.)